core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -37,6 +37,12 @@ object ReplicaAssignment {
   }
 }
 
+
+/**
+ * @param replicas the seq of brokers assigned to the partition
```

Comment:
Maybe worth mentioning that this list includes both the adding and removing replicas?

Reply:
Done.